Source/Python/WarpX.cpp

```diff
@@ -143,6 +143,10 @@ void init_WarpX (py::module& m)
             [](WarpX const & wx, int lev){ return wx.gett_new(lev); },
             py::arg("lev")
         )
+        .def("getdt",
+            [](WarpX const & wx, int lev){ return wx.getdt(lev); },
```

Comment:
This is fine for now, but there is a potential for out of bounds access if a bad lev is passed in. `getdt` is doing a simple array access, i.e. `dt[lev]`. You can add bounds checking in this interface, or `getdt` could be changed to use `dt.at(lev)` and then pybind11 would catch the `std::out_of_range` exception.